CHANGELOG.md

```diff
@@ -1,7 +1,41 @@
 # Changelog
 All notable changes to the COBOL Language Support extension are documented in this file.
 
-## [1.1.2](https://github.com/eclipse/che-che4z-lsp-for-cobol/compare/1.1.1...1.1.2) (2022-12-22)
+## [2.0.0](https://github.com/eclipse/che-che4z-lsp-for-cobol/compare/2.0.0...1.2.1) (2023-05-23)
+
+### Bug Fixes
+* Clear diagnostics when the changed content is not analysed
+* Issues with proc_grps json schema
+* Broken SSQ CASE statement grammar
+* Same copybook used more than once within a program shows error if copybook refers to another copybook #1833
+* Support repository paragraph
+* Tolerate json syntax
+* Enable ** match for .folders
+* DaCo extensionDependencies as per new naming convention.
+* Support SUPPRESS keyword in the copy statement #1816
```

Comment:
in previous releases we had links to commits for each line. Now we have only three here. Why?
Those link don't even work btw...

Reply:
The reason for that is the broken changelog generator. We have release tags on the master and development branches so it's impossible to generate commit history between this and previous release. This change log generated my filtering git commits history.

Reply:
Yesterday we had a meeting about this issue and agreed on practices to avoid it in future. The next release changelog should be in the better.